include/highfive/bits/H5Utils.hpp

```diff
@@ -45,6 +46,27 @@ inline std::vector<std::size_t> to_vector_size_t(const std::vector<std::size_t>&
     return vec;
 }
 
+// converter function for size_t -> hsize_t when size_t != hsize_t
+template <typename Size>
+inline std::vector<hsize_t> to_vector_hsize_t(const std::vector<Size>& vec) {
```

Comment:
This function exists in similar form here:
https://github.com/BlueBrain/HighFive/blob/7acd5bab14258f52f584e250c47358ed1477276a/include/highfive/bits/H5Slice_traits.hpp#L64